experiment_params/default.yaml

```diff
@@ -17,7 +17,9 @@ dataset:
   noise_std: 0
   radius_bound: [1.3, 2.3]
```

Comment:
Are these the default parameters of the paper?

Reply:
it depends on the environment. In the case of the pendulum it is. In the case of the spring it is (0.1, 1.0) ,etc.